include/exadg/aero_acoustic/volume_coupling.h

```diff
@@ -0,0 +1,123 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2023 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_AERO_ACOUSTIC_VOLUME_COUPLING_H_
+#define INCLUDE_EXADG_AERO_ACOUSTIC_VOLUME_COUPLING_H_
+
+#include <exadg/aero_acoustic/calculators/source_term_calculator.h>
+#include <exadg/aero_acoustic/single_field_solvers/acoustics.h>
+#include <exadg/aero_acoustic/single_field_solvers/fluid.h>
+#include <exadg/aero_acoustic/user_interface/parameters.h>
+
+namespace ExaDG
+{
+namespace AeroAcoustic
+{
+/**
+ * A class that handles the volume coupling between fluid and acoustic.
+ */
+template<int dim, typename Number>
+class VolumeCoupling
+{
+  using VectorType = dealii::LinearAlgebra::distributed::Vector<Number>;
+
+public:
+  void
+  setup(Parameters const &                           parameters_in,
+        std::shared_ptr<SolverAcoustic<dim, Number>> acoustic_solver_in,
+        std::shared_ptr<SolverFluid<dim, Number>>    fluid_solver_in)
+  {
+    parameters      = parameters_in;
+    acoustic_solver = acoustic_solver_in;
+    fluid_solver    = fluid_solver_in;
+
+    acoustic_solver_in->pde_operator->initialize_dof_vector_pressure(source_term_acoustic);
+    fluid_solver_in->pde_operator->initialize_vector_pressure(source_term_fluid);
```

Comment:
It would be good to unify these namings. Of course, something that is independent of the present PR.